x-pack/plugins/apm/public/components/app/diagnostics/apm_documents_tab.tsx

```diff
@@ -159,3 +159,49 @@ export function DiagnosticsApmDocuments() {
     </>
   );
 }
+
+function IntervalDocCount({
+  interval,
+}: {
+  interval?: {
+    metricDocCount: number;
+    eventDocCount: number;
+  };
+}) {
+  if (interval === undefined) {
+    return <>-</>;
+  }
+
+  return (
+    <EuiToolTip
+      content={`${asInteger(interval.metricDocCount)} docs / ${asInteger(
+        interval.eventDocCount
+      )} events`}
+    >
+      <div style={{ cursor: 'pointer' }}>
+        {asBigNumber(interval.metricDocCount)}&nbsp;
+        <EuiText
+          css={{ fontStyle: 'italic', fontSize: '80%', display: 'inline' }}
+        >
+          ({asBigNumber(interval.eventDocCount)} events)
+        </EuiText>
+      </div>
+    </EuiToolTip>
+  );
+}
+
+function asBigNumber(value: number): string {
```

Comment:
Can we co-locate the formatter with the other ones we have in `utils/formatters` ?

Reply:
Sure!